sorc/global_cycle.fd/cycle.f90

```diff
@@ -1,6 +1,8 @@
 !> @file
```

Comment:
There is a big legacy documentation block starting at line 270 in the code. This needs to be transfered to the doxygen documentation. I suggest you just cut it and put it in the program documentation.

Format tables like this:

```
hd1 | hd2 | hd3
------|-------|------
text in col 1 | col2 | col3
more col1 | more col2 | col3


```

Reply:
Done.